### PR TITLE
Updated the name of table creation function 

### PR DESCRIPTION
**Pull request by voremargot**

### CODE REVIEW

File: tests/test_tables.py [hunk 1]
```diff
@@ -2,70 +2,70 @@
 from pytest import raises
 from strapPy.bootstrap import bootstrap_distribution
 from strapPy.bootstrap import calculate_boot_stats
-from strapPy.data_vis import summary_tables
+    from strapPy.data_vis import create_tables
 
 def test_table_outputs():
     """Tests the functionality of the summary table function."""
```
Comment (BooleanJulien) [inline tests/test_tables.py]:
```suggestion
    """Tests the functionality of the create_tables function."""
```

File: tests/test_tables.py [hunk 2]
```diff
@@ -2,70 +2,70 @@
...
     st = calculate_boot_stats(np.random.randint(1, 20, 20), 1000,
                               level=0.95, random_seed=123, n=10,
                               estimator="median")
-    s, bs = summary_tables(st)
+    s, bs = create_tables(st)
     assert ("Sample median" in s.data.columns), "Test Statistic name incorrect"
     
     
 def test_table_errors():
     "Tests the functionality of the summary table Raise Error statements."
```
Comment (BooleanJulien) [inline tests/test_tables.py]:
```suggestion
    "Tests the functionality of the create_tables Raise Error statements."
```

**Review by BooleanJulien — APPROVED** [review]
Couple minor changes otherwise looks good

File: src/strapPy/data_vis.py [hunk 3]
```diff
@@ -81,7 +81,7 @@ def histogram_ci_plot(sample, rep, bin_size = 30, n="auto", ci_level=0.95,
     
 
 
-def summary_tables(stat, precision=2, estimator=True, alpha=True):
+def create_tables(stat, precision=2, estimator=True, alpha=True):
     """Makes two tables that summerize the statistics from the bootstrapped
```
Comment (gauthampughazhendhi) [inline src/strapPy/data_vis.py]:
```suggestion
    """Makes two tables that summarize the statistics from the bootstrapped 
```

File: src/strapPy/data_vis.py [hunk 4]
```diff
@@ -81,7 +81,7 @@ def histogram_ci_plot(sample, rep, bin_size = 30, n="auto", ci_level=0.95,
     
 
 
-def summary_tables(stat, precision=2, estimator=True, alpha=True):
+def create_tables(stat, precision=2, estimator=True, alpha=True):
```
Comment (gauthampughazhendhi) [inline src/strapPy/data_vis.py]:
```suggestion
def tabulate_stats(stat, precision=2, estimator=True, alpha=True):
```

@voremargot I personally feel tabulate_stats summarizes the function well but I am open to your feedback.

File: tests/test_tables.py [hunk 5]
```diff
@@ -2,70 +2,70 @@
 from pytest import raises
 from strapPy.bootstrap import bootstrap_distribution
 from strapPy.bootstrap import calculate_boot_stats
-from strapPy.data_vis import summary_tables
+    from strapPy.data_vis import create_tables
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
    from strapPy.data_vis import tabulate_stats
```

File: tests/test_tables.py [hunk 6]
```diff
@@ -2,70 +2,70 @@
...
 
 def test_table_outputs():
     """Tests the functionality of the summary table function."""
     
     st = calculate_boot_stats(np.random.randint(1, 20, 20), 1000,
                               level=0.95, random_seed=123)
-    s, bs = summary_tables(st)
+    s, bs = create_tables(st)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
    s, bs = tabulate_stats(st)
```

File: tests/test_tables.py [hunk 7]
```diff
@@ -2,70 +2,70 @@
...
     assert s.data.shape[0] == 1, "Stats table should have 1 row"
     assert s.data.shape[1] == 5, "Stats table should have 5 columns"
     assert bs.data.shape[0] == 1, "Parameter table should have 1 row"
     assert bs.data.shape[1] == 3, "Parameter table should have 3 columns"
     
-    s, bs = summary_tables(st, estimator=False, alpha=False)
+    s, bs = create_tables(st, estimator=False, alpha=False)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
    s, bs = tabulate_stats(st, estimator=False, alpha=False)
```

File: tests/test_tables.py [hunk 8]
```diff
@@ -2,70 +2,70 @@
...
     assert s.data.shape[0] == 1, "Stats table should have 1 row"
     assert s.data.shape[1] == 3, "Stats table should have 3 columns"
     assert bs.data.shape[0] == 1, "Parameter table should have 1 row"
     assert bs.data.shape[1] == 3, "Parameter table should have 3 columns"
     
-    s, bs = summary_tables(st, alpha=False)
+    s, bs = create_tables(st, alpha=False)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
    s, bs = tabulate_stats(st, alpha=False)
```

File: tests/test_tables.py [hunk 9]
```diff
@@ -2,70 +2,70 @@
...
     assert s.data.shape[0] == 1, "Stats table should have 1 row"
     assert s.data.shape[1] == 4, "Stats table should have 4 columns"
     assert bs.data.shape[0] == 1, "Parameter table should have 1 row"
     assert bs.data.shape[1] == 3, "Parameter table should have 3 columns"
     
     st = calculate_boot_stats(np.random.randint(1, 20, 20), 1000,
                               level=0.95, random_seed=123, n=10)
-    s, bs = summary_tables(st)
+    s, bs = create_tables(st)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
    s, bs = tabulate_stats(st)
```

File: tests/test_tables.py [hunk 10]
```diff
@@ -2,70 +2,70 @@
...
     assert bs.data.shape[0] == 1, "Parameter table should have 1 row"
     assert bs.data.shape[1] == 4, "Parameter table should have 4 columns"
     
     st = calculate_boot_stats(np.random.randint(1, 20, 20), 1000,
                               level=0.95, random_seed=123, n=10,
                               estimator="median")
-    s, bs = summary_tables(st)
+    s, bs = create_tables(st)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
    s, bs = tabulate_stats(st)
```

File: tests/test_tables.py [hunk 11]
```diff
@@ -2,70 +2,70 @@
...
     assert ("Sample median" in s.data.columns), "Test Statistic name incorrect"
     
     
 def test_table_errors():
     "Tests the functionality of the summary table Raise Error statements."
     
     with raises(TypeError) as e:
-        summary_tables(6, precision=2, estimator=True, alpha=True)
+        create_tables(6, precision=2, estimator=True, alpha=True)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
        tabulate_stats(6, precision=2, estimator=True, alpha=True)
```

File: tests/test_tables.py [hunk 12]
```diff
@@ -2,70 +2,70 @@
...
     assert str(e.value) == ("The stats parameter must be created from \
 calculate_boot_stats() function.")
     
     st = calculate_boot_stats(np.random.randint(1, 20, 20), 1000,
                               level=0.95, random_seed=123)
     with raises(TypeError) as e:
-        summary_tables(st, precision=2.9, estimator=True, alpha=True)
+        create_tables(st, precision=2.9, estimator=True, alpha=True)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
        tabulate_stats(st, precision=2.9, estimator=True, alpha=True)
```

File: tests/test_tables.py [hunk 13]
```diff
@@ -2,70 +2,70 @@
...
     assert str(e.value) == "The precision parameter must be of type int."
     
     with raises(TypeError) as e:
-        summary_tables(st, precision=2, estimator="Y", alpha=True)
+        create_tables(st, precision=2, estimator="Y", alpha=True)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
        tabulate_stats(st, precision=2, estimator="Y", alpha=True)
```

File: tests/test_tables.py [hunk 14]
```diff
@@ -2,70 +2,70 @@
...
     assert str(e.value) == ("The estimator and alpha parameters must be \
 of type boolean.")
     
     with raises(TypeError) as e:
-        summary_tables(st, precision=2, alpha=9)
+        create_tables(st, precision=2, alpha=9)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
        tabulate_stats(st, precision=2, alpha=9)
```

File: tests/test_tables.py [hunk 15]
```diff
@@ -2,70 +2,70 @@
...
     assert str(e.value) == ("The estimator and alpha parameters must be \
 of type boolean.")
     
     del st["lower"]
     with raises(TypeError) as e:
-        summary_tables(st, precision=2)
+        create_tables(st, precision=2)
```
Comment (gauthampughazhendhi) [inline tests/test_tables.py]:
```suggestion
        tabulate_stats(st, precision=2)
```

File: README.md [hunk 16]
```diff
@@ -22,7 +22,7 @@ $ pip install -i https://test.pypi.org/simple/ strappy
 - **bootstrap_distribution:** A sampling distribution of `rep` replicates is generated for a specified estimator with replacement for a given bootstrap sample size.  
 - **calculate_boot_stats:** Calculates a confidence interval for a given sampling distribution as well as other bootstapped statistics.  
 - **histogram_ci_plot:** Makes a histogram of a boostrapped sampling distribution with its confidence interval and oberserved sample statistic.  
-- **summary_tabels:** Generates a table that contains a given sampling distribution's mean and standard deviation along with relevant statistics as well as a summary table of the bootstrap distributions parameters  
+- **create_tables:** Generates a table that contains a given sampling distribution's mean and standard deviation along with relevant statistics as well as a summary table of the bootstrap distributions parameters
```
Comment (gauthampughazhendhi) [inline README.md]:
```suggestion
- **tabulate_stats:** Generates a table that contains a given sampling distribution's mean and standard deviation along with relevant statistics as well as a summary table of the bootstrap distributions parameters  
```

**Review by gauthampughazhendhi — CHANGES_REQUESTED** [review]
@voremargot Please find my suggestions

**Comment by voremargot** [comment]
@gauthampughazhendhi  I renamed the function tabulate_stats. I did it in the .py files and not through github just to make sure I identified all the instances of create_tables. I also resolved merge conflicts with the dev branch.

**Comment by gauthampughazhendhi** [comment]
Looks good to me

**Pull request merged**